### DISCUSSION TITLE
coder_app in multiple containers

### DISCUSSION THREAD

**Discussion by giovanidiniz** [coder/coder#8119] [General]
Hi, there,

I've faced with this situation in which, ideally, I would have different apps running in different containers, but I can't figure if that's a possible in coder, and, if so, how to configure my terraform files to do that. 

As a dumb example, say, for whatever reason, I needed to run two different containers each with code-server running in them. What would the .tf file look like in that case?

**Comment by sharkymark** [comment]
I have a Kubernetes pod example but it can be done with Docker.

You add more than one container section in the pod spec, and since they all talk on `localhost` you can use one `coder_agent` and have multiple `coder_apps` with the specific ports of the IDEs you want.

https://github.com/sharkymark/v2-templates/tree/main/k8s-multi-container
